packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -227,10 +227,12 @@ export default class ListControl extends React.Component {
     this.setState({ itemsCollapsed: updatedItemsCollapsed });
   };
 
-  renderItem = (item, index) => {
+  renderItem = (item, index, list) => {
     const { field, classNameWrapper, editorControl, resolveWidget } = this.props;
     const { itemsCollapsed } = this.state;
     const collapsed = itemsCollapsed.get(index);
+    const minimum = field.get('minimum');
+    const allowRemove = typeof minimum === 'number' ? list.size > minimum : true;
```

Comment:
If we're going to block removal to enforce a minimum, we should load with the minimum number of blank list items in place.

Reply:
@erquhart What's the right way to do that ?
Do we already have a place where we handle this kind of logic ?